example/database/migrations/2020_01_30_0000_create_books_table.php

```diff
@@ -13,7 +13,7 @@ class CreateBooksTable extends Migration
      */
     public function up()
     {
-        Schema::create('books', function (Blueprint $table) {
+        Schema::create('books', static function (Blueprint $table) {
```

Comment:
There are still lots of changes where you added `static`. Please remove all of them where you added them.

Reply:
Do you use the PhpStorm Plugin https://plugins.jetbrains.com/plugin/7622-php-inspections-ea-extended- ? This plugin told me that static methods are better. As long as you don't use `$this` in the closure, `static` should be used. 

Sources: 
* https://github.com/slevomat/coding-standard/issues/400
* https://blobfolio.com/2017/09/25-easy-php7-micro-optimizations/ (Use Static Methods and Properties
)

Reply:
Yeah I use this package but I don't use the static keyword for closures. I know there might be a small improvement when using, but I don't think it is noticeable and I like my code cleaner and without them.

Reply:
ok fine.